packages/mobile/src/home/__snapshots__/NotificationBox.test.tsx.snap

```diff
@@ -63,58 +63,30 @@ exports[`NotificationBox renders correctly for with all notifications 1`] = `
               }
             }
           >
-            <View
```

Comment:
The change in this snapshot is that the backup account key notification now shows up first, before the invite and payment requests notification. 

This backup notification only shows up for 24 hours anyways because after that backing up the account key is mandatory.